lib/rubocop/options.rb

```diff
@@ -241,14 +241,18 @@ def validate_compatibility
       unless boolean_or_empty_cache?
         raise ArgumentError, '-C/--cache argument must be true or false'
       end
-      validate_auto_gen_config
-      validate_parallel
+      validate_options
 
       return if incompatible_options.size <= 1
       raise ArgumentError, 'Incompatible cli options: ' \
                            "#{incompatible_options.inspect}"
     end
 
+    def validate_options
```

Comment:
I don't think this method adds much, so it might be better to just disable the length check for this method.